### PR TITLE
Updates for the 0 version

### PR DESCRIPTION
**Pull request by gbittmann**
The 0 version is still taking updates, but has been moved.

### CODE REVIEW

File: files/angular.ui-router/update.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "packageManager": "github",
   "name": "angular.ui-router",
-  "repo": "angular-ui/ui-router",
+  "repo": "angular-ui/ui-router/tree/legacy",
```
Comment (jimaek) [inline files/angular.ui-router/update.json]: `/tree/legacy` is not part of the repo name, its a path inside the `angular-ui/ui-router` repo. So this is incorrect

**Comment by gbittmann** [comment]
Thanks for helping @jimaek. Does this mean updates will need to be done manually? It's currently at 0.3.0, but the new versions aren't getting auto updated.

**Comment by jimaek** [comment]
If the old versions are tagged correctly and have the same file structure then nothing is needed. The bot will keep all major versions updated. 0.x and 1.x

**Comment by gbittmann** [comment]
@jimaek thanks for your continued help. I looked at the tags / releases. I'm not sure what would be wrong, but I do see several releases that have not been picked up but are months old. [https://github.com/angular-ui/ui-router/releases](https://github.com/angular-ui/ui-router/releases)
If there is something wrong on the ui-router project, I can open an issue there to see if they can fix it just let me know.

**Comment by jimaek** [comment]
@MartinKolarik Any idea why the bot is not updating 0.x?

**Comment by MartinKolarik** [comment]
@jimaek yeah, [we don't download](https://github.com/jsdelivr/libgrabber/blob/master/lib/project.js#L40) `0.x` releases if there already is a `1.x`. I think the reasoning behind that was that `0.x` are like beta releases and there's no point adding them if there already is a stable release.

**Comment by gbittmann** [comment]
@MartinKolarik Is there any way to change that behavior in this case? Version 1 is alpha and the 0 version is stable and taking bug fixes.

**Comment by MartinKolarik** [comment]
We could remove that check, but since there's a difference in file structure between 0 and 1, auto-updates will only work for one of them.

**Comment by gbittmann** [comment]
I would think the 0 version would be the best fit for now since it's the stable version if you have to choose one, but I am biased. Could you go by usage metrics?

**Comment by christopherthielen** [comment]
ui-router member here.

0.x releases are still the "stable" releases.  1.0 releases should be finalized in a few months.

github.com/angular-ui/ui-router is no longer the right place to be pulling releases from.  Can jsdelivr use npm packages, or github only?

If github only, it should be fetched from our bower releases at http://github.com/angular-ui/angular-ui-router-bower which should have the same package structure as before.


---

We have changed our distribution to allow angular 1 and angular 2 code to co-exist in a single github repository.   From the single monolithic repository, we're publishing 3 different packages:

- ui-router-core
- ui-router-ng2 (angular 2)
- angular-ui-router (angular 1)

We tag the overall repository when releasing, but we no longer publish directly from the tag.  We now build npm packages and bower packages and publish separately from the repository.

**Pull request closed without merging**